Recent change removed methods required by elastic/go-sysinfo with no version number bump

Related to: https://github.com/prometheus/procfs/issues/167

This commit removed methods that are used by other libraries (breaking changes) while maintaining v0.0.1 as the latest "release":
https://github.com/prometheus/procfs/commit/985b82318ecc7741179e9c11a99095cd1f6965c9#diff-33838b2a8ea2f8e4311d86efea5944c5L151

I propose that this project follow semantic versioning. In the meantime, would it be possible to add the methods required by elastic/go-sysinfo back?
https://github.com/elastic/go-sysinfo/blob/master/providers/linux/boottime_linux.go

These are the specific errors:
```
../../../github.com/elastic/go-sysinfo/providers/linux/boottime_linux.go:40:17: fs.NewStat undefined (type procfs.FS has no field or method NewStat)
../../../github.com/elastic/go-sysinfo/providers/linux/host_linux.go:75:23: h.procFS.NewStat undefined (type procFS has no field or method NewStat)
../../../github.com/elastic/go-sysinfo/providers/linux/host_linux.go:93:17: fs.NewStat undefined (type procFS has no field or method NewStat)
../../../github.com/elastic/go-sysinfo/providers/linux/process_linux.go:49:23: s.procFS.NewProc undefined (type procFS has no field or method NewProc)
../../../github.com/elastic/go-sysinfo/providers/linux/process_linux.go:82:19: p.fs.NewProc undefined (type procFS has no field or method NewProc)
```

There was a partial revert of the breaking changes for the sake of client_golang here:
https://github.com/prometheus/procfs/commit/9b1831cbb3bb0a76227ba0e19bdfeba868679544

It seems the re-addition of the func `NewStat` here was meant to be bound to `(fs FS)`:
https://github.com/prometheus/procfs/commit/9b1831cbb3bb0a76227ba0e19bdfeba868679544#diff-33838b2a8ea2f8e4311d86efea5944c5R148

--- a/proc.go
+++ b/proc.go
@@ -79,6 +79,13 @@ func (fs FS) Self() (Proc, error) {
 	return fs.Proc(pid)
 }
 
+// NewProc returns a process for the given pid.
+//
+// Deprecated: use fs.Proc() instead
+func (fs FS) NewProc(pid int) (Proc, error) {
+	return fs.Proc(pid)
+}
+
 // Proc returns a process for the given pid.
 func (fs FS) Proc(pid int) (Proc, error) {
 	if _, err := os.Stat(fs.proc.Path(strconv.Itoa(pid))); err != nil {

--- a/stat.go
+++ b/stat.go
@@ -153,6 +153,14 @@ func NewStat() (Stat, error) {
 	return fs.Stat()
 }
 
+// NewStat returns information about current cpu/process statistics.
+// See https://www.kernel.org/doc/Documentation/filesystems/proc.txt
+//
+// Deprecated: use fs.Stat() instead
+func (fs FS) NewStat() (Stat, error) {
+	return fs.Stat()
+}
+
 // Stat returns information about current cpu/process statistics.
 // See https://www.kernel.org/doc/Documentation/filesystems/proc.txt
 func (fs FS) Stat() (Stat, error) {

